package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
Didn't you checkout the difference from the 1.3.6 to the head was the dependency of the socket.io-client?